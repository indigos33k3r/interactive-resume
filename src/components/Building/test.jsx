import React from 'react'
import {shallow} from 'enzyme'

describe('Building', () => {
  it('renders <Building />', () => {
    const component = render()

    expect(component).toMatchSnapshot()
  })

  function render() {
    const defaultProps = {
      buildingClass: 'welcome',
      children: 'Some child text'
    }
    const Building = require('.').default

    return shallow(<Building {...defaultProps} />)
  }
})
