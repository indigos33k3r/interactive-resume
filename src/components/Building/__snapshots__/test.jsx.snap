// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Building renders <Building /> 1`] = `
<section>
  <div
    className="building building--welcome"
  >
    Some child text
  </div>
</section>
`;
